### DISCUSSION TITLE
Is it possible to dynamically change the data of react-table?

### DISCUSSION THREAD

**Discussion by bharathch0404** [TanStack/table#2182] [General]
I want to change the table dynamically by changing the data, for example adding or removing data. I couldn't find an example for the same. Was wondering if this is possible.?

**Comment by jstacoder** [comment]
i as well am looking for a way to load the initial table, then to load more data after the table renders.

**Comment by gargroh** [comment]
Yeah...just change the reference of data prop, table will rerender with
updated data
